fontforge_font_creator/creator.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+import fontforge
+import os
+import ruamel.yaml
+import shutil
+
+
+class FontMaker:
+    """
+    Usage Example:
+        fm = FontMaker('osmaxx_v1_definition.yml')
+        fm.create_font()
+    """
+    def __init__(self, font_config):
+        self.config = self.yml_reader(font_config)
+        self._base_path = '/home/fonts/osmaxx/svg'
+        self.n = fontforge.font()
```

Comment:
Is `n` an established naming convention/idiom for fontforge usage, so that a different name would confuse readers familiar with fontforge? If not, please use a more speaking name for this instance variable.